sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -420,7 +420,7 @@ def run_pipeline(self, pipeline, options, pipeline_proto=None):
            for pcoll in self.proto_pipeline.components.pcollections.values()):
       if (not options.view_as(StandardOptions).streaming and
           not options.view_as(DebugOptions).lookup_experiment(
-              'allow_unbounded_pcollections_in_batch_mode')):
+              'unsafely_attempt_to_process_unbounded_data_in_batch_mode ')):
```

Comment:
There is a tiny but extra space at the end of the string which is causing the internal tests unhappy